infra/gcb/build_project.py

```diff
@@ -384,14 +399,28 @@ def run_build(build_steps, project_name, tag):
   print(build_id)
 
 
+# pylint: disable=missing-function-docstring
 def main():
   if len(sys.argv) != 2:
     usage()
 
+  gcp_project = 'oss-fuzz'
+  gcp_project_base = 'oss-fuzz-base'
   project_dir = sys.argv[1].rstrip(os.path.sep)
-  steps = get_build_steps(project_dir)
-
+  dockerfile_path = os.path.join(project_dir, 'Dockerfile')
+  project_yaml_path = os.path.join(project_dir, 'project.yaml')
   project_name = os.path.basename(project_dir)
+
+  with open(dockerfile_path) as dockerfile:
+    dockerfile_contents = dockerfile.readlines()
+
+  with open(project_yaml_path) as project_yaml_file:
```

Comment:
Let's bring back load_project_yaml for convenience. That can call set_yaml_defaults.